applications/luci-app-openvpn/luasrc/model/cbi/openvpn.lua

```diff
@@ -126,5 +125,8 @@ function proto.cfgvalue(self, section)
 	return val or "udp"
 end
 
+function m.on_after_commit(self,map)
+	require("luci.sys").call('/sbin/reload_config')
```

Comment:
Please don't do that, it might have unintended side effects. Better invoke reload on the openvpn init script instead.